bin/node-template/runtime/src/lib.rs

```diff
@@ -127,6 +127,8 @@ parameter_types! {
 	/// We allow for 2 seconds of compute with a 6 second average block time.
 	pub const MaximumBlockWeight: Weight = 2 * WEIGHT_PER_SECOND;
 	pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
+	/// Assume 10% of weight for average on_initialize calls.
+	pub const MaximumExtrinsicWeight: Weight = Perbill::from_percent(75 - 10) * MaximumBlockWeight::get();
```

Comment:
Nit: either this should directly use `AvailableBlockRatio`, or, more realistically, the comment should mention that this `75` comes from it.